examples/all-clusters-app/esp32/main/CMakeLists.txt

```diff
@@ -39,6 +39,7 @@ set(SRC_DIRS_LIST
                       "${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/src/app/clusters/administrator-commissioning-server"
                       "${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/src/app/clusters/test-cluster-server"
                       "${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/src/app/clusters/descriptor"
+                      "${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/src/app/clusters/drlc-server"
```

Comment:
does ESP32 will use something similar to DRLC.cpp ? , right now you just declare the server sources but it is not used within ESP app?